apis.py

```diff
@@ -29,17 +29,23 @@ def get_coins():
 def get_pool_size(pool, coin):
     if pool not in CACHED_POOL_SIZES or time.time() > CACHED_POOL_SIZES[pool][0] + API_WAIT_SECONDS:
         try:
-            api_key = CONFIG['pools'][pool]['api_key']
-            r = requests.get(
-                CONFIG['pools'][pool]['coins'][coin]['pool_api']['url'].replace('$API_KEY', api_key)
-            ).json()
+            request = CONFIG['pools'][pool]['coins'][coin]['pool_api']['url']
+            if contains_api_key(request):
+                try:
+                    api_key = USER_CONFIG['pools'][pool]['api_key']
+                    request.replace('$API_KEY', api_key)
+                except Exception:
+                    print(pool, 'API key was not present for request:', request)
```

Comment:
Do we want to just swallow this exception? Seems like having $API_KEY in the request but then failing to find it indicates an error in the config file that should be addressed and not ignored.

Reply:
Aren't we doing that already with L43-L44?This way the print message at least tells us what's wrong.

Reply:
True, maybe we should limit L43 to only errors with the request instead of a catch-all